src/main/java/com/wiley/autotest/selenium/elements/upgrade/v3/NullShould.java

```diff
@@ -0,0 +1,81 @@
+package com.wiley.autotest.selenium.elements.upgrade.v3;
+
+import com.wiley.autotest.selenium.SeleniumHolder;
+import com.wiley.autotest.selenium.context.SearchStrategy;
+import com.wiley.autotest.selenium.elements.upgrade.NullTeasyElement;
+import com.wiley.autotest.selenium.elements.upgrade.TeasyElement;
+import com.wiley.autotest.selenium.elements.upgrade.TeasyElementData;
+import com.wiley.autotest.selenium.elements.upgrade.v3.conditions.element.*;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+import java.util.function.Function;
+
+/**
+ * Assertions for Null Element
+ */
+public class NullShould implements Should {
+
+    private TeasyFluentWait<WebDriver> fluentWait;
+    private TeasyElementData elementData;
+    private TeasyElementFinder finder;
+
```

Comment:
empty lines

Reply:
daaaaaym, I need to set up my code formatting rules better 👍